src/analysis/classic/priest/shared/lowRankSpells.ts

```diff
@@ -0,0 +1,14 @@
+import SPELLS from 'common/SPELLS/classic/druid'; // UPDATE THE CLASS
```

Comment:
```suggestion
import SPELLS from 'common/SPELLS/classic/priest';
```

should be priest, right?